manage-data/ingest/transform-enrich/common-mistakes.md

```diff
@@ -14,33 +14,33 @@ There are many ways to achieve similar results when creating ingest pipelines, w
 This guide does not provide guidance on optimizing for ingest pipeline performance.
 :::
 
-## If Statements
+## Write concise `if` statements
```

Comment:
Use grammatically parallel headings. In this case, I've updated them to all start with a verb.